scripts/setup-ci

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+git submodule update --init
+
+# Verify dependencies that can be installed with apt (if available)
+missing=()
+
+if ! which openocd >/dev/null; then
+    missing+=(openocd)    
+fi
+if ! which riscv64-unknown-elf-gcc >/dev/null; then
+    missing+=(gcc-riscv64-unknown-elf)
+fi
+if ! which yosys >/dev/null; then
+    missing+=(yosys)
+fi
+if ! which expect >/dev/null; then
+    missing+=(expect)
+fi
+if ! which ccache >/dev/null; then
+    missing+=(ccache)
+fi
+if ! which verilator >/dev/null; then
+    missing+=(verilator libevent-dev libjson-c-dev)
+fi
+
+if [ ${#missing[@]} -gt 0 ]; then
+    echo "Missing packages: ${missing[@]}" >&2
+    sudo apt install -y "${missing[@]}"
+fi
+
+# Warn about different versions
+
+if ! riscv64-unknown-elf-gcc --version | grep ' 8.3.' >/dev/null; then
```

Comment:
The toolchain I'm using has this version:

riscv64-unknown-elf-gcc (SiFive GCC 10.1.0-2020.08.2) 10.1.0

Maybe we need to update scripts/setup too.

Reply:
Oh, so 8.3 is good, 9.x is bad, but 10.1 is good again?   I'll have to check that out.